content/doc/developer/tutorial/create.adoc

```diff
@@ -36,37 +36,43 @@ Choose a number or apply filter (format: [groupId:]artifactId, case sensitive co
 Choose io.jenkins.archetypes:hello-world-plugin version:
 1: 1.1
 2: 1.2
-Choose a number: 2: *2* // <2>
+3: 1.3
+Choose a number: 3: *3* // <2>
```

Comment:
Note that I just released 1.3 and it will take until Sunday (IIRC) for the Maven Central index to reflect this in the archetypes catalog.

Reply:
I just tried it and it exists, so we can merge this now.